cmake/cpack_rpm.cmake

```diff
@@ -223,7 +223,7 @@ SETA(CPACK_RPM_server_PACKAGE_REQUIRES
   "MariaDB-client >= 10.6.1")
 
 IF(WITH_WSREP)
-  SETA(CPACK_RPM_server_PACKAGE_REQUIRES
+  SETA(CPACK_RPM_server_PACKAGE_RECOMMENDS
```

Comment:
@grooverdan, `"rsync" "lsof" "grep" "gawk" "iproute"
     "coreutils" "findutils" "tar"`
seem must stay in CPACK_RPM_server_PACKAGE_REQUIRES, right? 
Please, consider adding new line for CPACK_RPM_server_PACKAGE_RECOMMENDS.

Reply:
I think so too as they are needed several stuff in scripts.